### PR TITLE
Introduce new routes at applications: /switches and /tests

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

This introduces new routes in the [applications] app to serve new metadata JSON objects to be used by the client side (request from commercial dev). 

This is a work in progress and the first of possibly 3 PRs. The second will open the corresponding routes in the router config and the third might edit the objects if needed (before handing this over to the clients).

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No

### CODE REVIEW

**Comment by shtukas** [comment]
Platform update PR: https://github.com/guardian/platform/pull/1425

File: applications/app/controllers/Nx1ConfigController.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package controllers
+
+import play.api.libs.json.{Format, Json}
+import play.api.mvc.{Action, AnyContent, BaseController, ControllerComponents, RequestHeader}
+import views.support.CamelCase
+import experiments.ActiveExperiments
+import experiments.ActiveExperiments._
+
+/*
+  Nx1Config (project name) was introduced to provide the client-side with metadata in places where the
```
Comment (sndrs) [inline applications/app/controllers/Nx1ConfigController.scala:10]: :heart:

File: applications/conf/routes [hunk 2]
```diff
@@ -84,6 +84,10 @@ GET        /$path<theguardian|theobserver>/$year<\d\d\d\d>/$month<\w\w\w>/$day<\
 POST       /atom/quiz/:id/*path                                                 controllers.QuizController.submit(id, path)
 GET        /atom/youtube/:id.json                                               controllers.YoutubeController.getAtomId(id: String)
 
+# Nx1Config
```
Comment (sndrs) [inline applications/conf/routes:87]:
```suggestion
# Nx1Config
# mark: 2QJfZo - keep these in sync with other instances https://git.io/JkRox
```

File: dev-build/conf/routes [hunk 3]
```diff
@@ -370,6 +370,10 @@ GET            /$shortCode<p/[\w]+>/:campaignCode
 POST           /atom/quiz/:id/*path                                                                                              controllers.QuizController.submit(id,path)
 GET            /atom/youtube/:id.json                                                                                            controllers.YoutubeController.getAtomId(id: String)
 
+# Nx1Config
```
Comment (sndrs) [inline dev-build/conf/routes:373]:
```suggestion
# Nx1Config
# mark: 2QJfZo - keep these in sync with other instances https://git.io/JkRox
```

File: applications/app/controllers/Nx1ConfigController.scala [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  usual article client side JavaScript config object is not available
+
+  The urls currently active are
+
+    - https://www.theguardian.com/switches.json
+    - https://www.theguardian.com/tests.json
```
Comment (sndrs) [inline applications/app/controllers/Nx1ConfigController.scala:16]:
```suggestion
    - https://www.theguardian.com/tests.json
    - mark: 2QJfZo - keep these in sync with other instances https://git.io/JkRox
```

**Review by sndrs — APPROVED** [review]
very happy if rest of dotcom are – thank you!

**Pull request merged**